README.rst

```diff
@@ -429,10 +429,6 @@ Example result response (cut to one element only instead of 10):
 Filters
 -------
 
-**note**: this feature is compatible with django-filter up to version 0.15.
```

Comment:
I would leave something like: "this feature has been tested up to django-filter 1.0".